rebasehelper/application.py

```diff
@@ -450,6 +450,9 @@ def _prepare_rebased_repository(cls, patches, rebased_sources_dir):
         for patch in patches['applied']:
             shutil.copy(patch.path, rebased_sources_dir)
 
+        for patch in patches['not_applied']:
```

Comment:
How about joining the two loops: `for patch in patches['applied'] + patches['not_applied']:`?